src/lib/theme/colors.ts

```diff
@@ -139,6 +139,34 @@ const palette: Readonly<Record<string, Record<Shade, HexColor>>> = {
     950: '#461B02',
     DEFAULT: '#F8A208',
   },
+  rhino: {
+    50: '#f1f7fd',
+    100: '#e0edf9',
+    200: '#c7dff6',
+    300: '#a1cbef',
+    400: '#74afe6',
+    500: '#5392de',
+    600: '#3f77d1',
+    700: '#3564c0',
+    800: '#31519c',
+    900: '#2c467c',
+    950: '#273860',
+    DEFAULT: '#273860',
+  },
+  pigeonPost: {
```

Comment:
Same comment as above, although `pigeonPost` is 💯

Reply:
I _do_ hate the camel-case color names though.